cmd/outflux/migrate_i_test.go

```diff
@@ -212,3 +212,72 @@ func TestMigrateFieldsAsJson(t *testing.T) {
 		t.Errorf("expected time > %v and fields={\"field1\": 1}\ngot: time %s, field1=%s", start, time, fieldsCol)
 	}
 }
+
+func TestMigrateRenameOutputSchema(t *testing.T) {
+	//prepare influx db
```

Comment:
Seems this kind of thing is used in several functions, maybe extract to a helper function? `prepareTestInfluxDB()`?